Google.PowerShell.Tests/Compute/ManagedGceInstanceCmdletTests.cs

```diff
@@ -69,5 +69,82 @@ public void TestGetRegionGceManagedInstanceGroup()
             Assert.IsNotNull(secondGroup);
             Assert.AreEqual(secondGroup.Name, SecondTestGroup.Name);
         }
+
+        /// <summary>
+        /// Tests that Remove-GceManagedInstanceGroup works with -Region option.
+        /// </summary>
+        [Test]
+        public void TestRemoveGceManagedInstanceGroupByRegion()
+        {
+            string instanceGroupName = "instance-group";
+            Mock<RegionInstanceGroupManagersResource> instances =
+                  ServiceMock.Resource(s => s.RegionInstanceGroupManagers);
+            instances.SetupRequest(
+                  item => item.Delete(FakeProjectId, FakeRegionName, instanceGroupName), DoneOperation);
+
+            Pipeline.Commands.AddScript(
+                $"Remove-GceManagedInstanceGroup -Name {instanceGroupName} -Region {FakeRegionName}");
+            Pipeline.Invoke();
+
+            instances.VerifyAll();
+        }
+
+        /// <summary>
+        /// Tests that Remove-GceManagedInstanceGroup works when pipelining regional instance group.
+        /// </summary>
+        [Test]
+        public void TestRemoveGceManagedInstanceGroupPipelineRegional()
+        {
+            string instanceGroupName = "RegionalInstanceGroup";
+            InstanceGroupManager regionalInstanceGroup = new InstanceGroupManager()
+            {
+                Name = instanceGroupName,
+                Region = $"{ComputeHttpsLink}/projects/{FakeProjectId}/regions/{FakeRegionName}",
+                SelfLink = $"{ComputeHttpsLink}/projects/{FakeProjectId}/regions/{FakeRegionName}/instanceGroupManagers/{instanceGroupName}"
```

Comment:
This line is long and repeats the above line. Perhaps separate the repeated parts into a common variable?

    string regionLink = $"{ComputeHttpsLink}/projects/{FakeProjectId}/regions/{FakeRegionName}";
    ....
        Region = regionLink,
        SelfLink = $"{regionLink}/instanceGroupManagers/{instanceGroupName}"